_benchmark/user-guide/distributed-load.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: default
+title: Configuring OpenSearch Benchmark
+nav_order: 10
+parent: User guide
+---
+
+# Running distributed load generation 
+
+
+By default, OpenSearch Benchmark runs loads on the same machine where the benchmark started. However, you can generate addiional benchmarking load, particularly on large clusters with multiple machines, you can use multiple load drivers. This page details how to distribute benchmark loads across multiple machines in a single cluster.
+
+## System architecture 
+
+The following tutorial will use the following three-node architecture, each generated in [Amazon EC2](https://docs.aws.amazon.com/ec2/?nc2=h_ql_doc_ec2):
+
+- **Node 1**: Node 1 acts as the _coordinator node_ which will enable distribution and communication between the other two nodes.
+- **Node 2** and **Node 3**: The remaining nodes in the cluster are used to generate load for the benchmark test.
+
+OpenSearch Benchmark must be installed on all nodes. For instructions on installing Benchmark, see [Installing OpenSearch Benchmark]({{site.url}}{{site.baseurl}}/benchmark/installing-benchmark/).
+
+Make note of each nodes IP addresses. This tutorial uses the following IP addresses for each node:
+
+**Node 1 - Coordinator node**: 192.0.1.0
+**Node 2 - Worker node**: 198.52.100.0
+**Node 3 - Worker node**: 198.53.100.0
+
+## Step 1: Enabling each node for communication
+
+Make sure to enable each node to communicate with each other. In the AWS Console:
+
+1. Go to the EC2 host for the node.
+2. Select **Security**, then the security group associated with the node. 
+3. Use **Add inbound rules** to open traffic to the node, based on the Port Range and traffic type of your cluster.
+
+## Step 2: Running daemon processes on each node
+
+Now, start OpenSearch Benchmark on each node, using the `--node-ip` to intailize OpenSearch Benchmark on the node itself, then `--coordinator-ip` to connect each node to the Coordinator node.
+
+For **Node 1**, the following command identifies the node as coordinator:
+
+```
+opensearch-benchmarkd start —node-ip=192.0.1.0 —coordinator-ip=192.0.1.0
```

Comment:
Need to ensure that the parameters use `--` for the beginning. For example, `—node-ip` should be `--node-ip` and `—coordinator-ip` should be `--coordinator-ip`.